File: src/main/java/org/shark/example/dao/repository/pojo/JsonExampleDo.java
package org.shark.example.dao.repository.pojo;


import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "JSON_EXAMPLE")
@Data
public class JsonExampleDo {
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    @Column(name = "ID")
    private Integer id;

    @Column(name = "JSON_DATA")
    private String jsonData;
}

File: src/main/kotlin/org/shark/example/dao/repository/pojo/JsonExampleDo.kt
package org.shark.example.dao.repository.pojo

import jakarta.persistence.*

@Entity
@Table(name = "JSON_EXAMPLE")
data class JsonExampleDo(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ID")
    var id: Int? = null,

    @Column(name = "JSON_DATA")
    var jsonData: String? = null
)
